tests/api_resources/test_issuer_fraud_record.py

```diff
@@ -0,0 +1,25 @@
+import stripe
+from tests.helper import StripeTestCase
+
+
+TEST_RESOURCE_ID = 'issfr_123'
+TEST_CHARGE_ID = 'ch_123'
+
+
+class IssuerFraudRecordTest(StripeTestCase):
+    def test_is_listable(self):
+        resources = stripe.IssuerFraudRecord.list()
+        self.assert_requested(
+            'get',
+            '/v1/issuer_fraud_records'
+        )
+        self.assertIsInstance(resources.data, list)
+        self.assertIsInstance(resources.data[0], stripe.IssuerFraudRecord)
+
+    def test_is_retrievable(self):
+        resource = stripe.IssuerFraudRecord.retrieve(TEST_RESOURCE_ID)
+        self.assert_requested(
+            'get',
+            '/v1/issuer_fraud_records/%s' % TEST_RESOURCE_ID
+        )
+        self.assertIsInstance(resource, stripe.IssuerFraudRecord)
```

Comment:
I removed the test case around CHARGE_ID since that should come as part of list's functionality, just like in the other libraries